mapFeatures.json

```
@@ -267,6 +267,7 @@
 		"quarry",
 		"railway",
 		"reservoir",
+                "residential",
```

```suggestion
		"residential",
```

Fix the wrong indentation